core/neat/mixins/_grid-column.scss

```diff
@@ -0,0 +1,39 @@
+@charset "UTF-8";
+/// Creates Neat a grid column of requested size.
+///
+/// @argument {number (unitless)} $width [1]
+///
+/// @argument {map} $grid [$neat-default-grid]
+///   The type of grid for this column. By default, the global
+///   `$neat-default-grid` will be used.
```

Comment:
:+1: I think that works @whmii. If we're consistent, it should become muscle-memory to think of `The grid` as the map you used to get it.